<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_empresa_datos</class>
 <widget class="QDialog" name="Form_empresa_datos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>551</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>81</red>
        <green>1</green>
        <blue>1</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>103</green>
        <blue>103</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>138</red>
        <green>138</green>
        <blue>138</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>81</red>
        <green>1</green>
        <blue>1</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>81</red>
        <green>1</green>
        <blue>1</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>81</red>
        <green>1</green>
        <blue>1</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>103</green>
        <blue>103</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>138</red>
        <green>138</green>
        <blue>138</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>81</red>
        <green>1</green>
        <blue>1</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>81</red>
        <green>1</green>
        <blue>1</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>81</red>
        <green>1</green>
        <blue>1</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>231</red>
        <green>231</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>103</green>
        <blue>103</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>138</red>
        <green>138</green>
        <blue>138</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>81</red>
        <green>1</green>
        <blue>1</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>81</red>
        <green>1</green>
        <blue>1</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="HighlightedText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>207</red>
        <green>207</green>
        <blue>207</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Alta Empresa</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 10pt &quot;KacstOne&quot;;
selection-background-color: rgb(255, 170, 127);
color: rgb(0, 0, 0);
background-color: rgb(81, 1, 1);
selection-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(136, 3, 3);

</string>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset>
        <normaloff>Íconos/hombre-en-el-escritorio-de-la-oficina-con-un-ordenador.png</normaloff>Íconos/hombre-en-el-escritorio-de-la-oficina-con-un-ordenador.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Empresa</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(239, 235, 231);
</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="icon">
           <iconset>
            <normaloff>Íconos/tarjeta-personal-de-datos-de-contacto.png</normaloff>Íconos/tarjeta-personal-de-datos-de-contacto.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Datos de la Empresa</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="acceptDrops">
              <bool>true</bool>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QGroupBox{
background-color: rgb(239, 235, 231);
border-style: solid;
border-width: 5px;
border-color: rgb(172, 55, 26);
}

QLabel{
background-color: rgb(239, 235, 231);
}

QLineEdit{
background-color: rgb(255, 255, 255);
}

QTextEdit{
background-color: rgb(255, 255, 255);
}

QComboBox{
background-color: rgb(255, 255, 255);
}

QDateEdit{
background-color: rgb(255, 255, 255);
}

</string>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Nombre de la Empresa:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lne_nombre">
                  <property name="font">
                   <font>
                    <family>KacstOne</family>
                    <pointsize>10</pointsize>
                    <weight>9</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Ciudad:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lne_ciudad">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Dirección:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lne_direccion">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Teléfono/Fax:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lne_telefono">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="4" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>E-mail:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lne_email">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="5" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Website:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lne_website">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="6" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Tipo de IVA:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cbx_tipo_iva">
                  <item>
                   <property name="text">
                    <string>Responsable inscripto</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Exento</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Monotributo</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="7" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Ingresos Brutos:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lne_ingresos_brutos">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="8" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>N° de CUIT:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lne_cuit">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="9" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>Nombre de  fantasia</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lne_nombre_fantasia">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="boton_limpiar">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(251, 204, 193);
</string>
       </property>
       <property name="text">
        <string>Limpiar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boton_guardar">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(251, 204, 193);
</string>
       </property>
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_nombre</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_ciudad</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_direccion</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_telefono</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_email</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_website</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_ingresos_brutos</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_cuit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_nombre_fantasia</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
